### PR TITLE
Add Casertavecchia (Italy)

### PR DESCRIPTION
**Pull request by twemyss**
This pull request adds basic details on the village of Casertavecchia (Italy). It also updates the README page for Europe to have a link to the village.

Answers UCL-RITS/rse-classwork-2020#4

### CODE REVIEW

File: europe/README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # EUROPE
+
```
Comment (dpshelio) [inline europe/README.md:2]: Hey! you forgot to add the country first, i.e., italy and its file, `italy.md`

File: europe/README.md [hunk 2]
```diff
@@ -1 +1,3 @@
...
+- [Casertavecchia](/europe/italy/casertavecchia.md)
```
Comment (dpshelio) [inline europe/README.md:3]:
```suggestion
- [Casertavecchia](./italy/casertavecchia.md)
```

**Review by dpshelio — CHANGES_REQUESTED** [review]
Please, update with the suggested changes.

**Comment by twemyss** [comment]
Thanks David - I have added the country README as requested, and updated the links accordingly.

**Review by dpshelio — APPROVED** [review]
👏

**Pull request merged**